client/my-sites/domains/domain-management/transfer/transfer-out/unlocked.jsx

```diff
@@ -11,28 +12,65 @@ import Card from 'components/card';
 import SectionHeader from 'components/section-header';
 import { getSelectedDomain } from 'lib/domains';
 import Button from 'components/button';
-import { requestTransferCode, enableDomainLocking } from 'lib/upgrades/actions';
+import { requestTransferCode, cancelTransferRequest } from 'lib/upgrades/actions';
 import notices from 'notices';
 import { displayRequestTransferCodeResponseNotice } from './shared';
 import support from 'lib/url/support';
 
 class Unlocked extends React.Component {
-	state = {
-		submitting: false
-	};
+	constructor( props ) {
```

Comment:
Yup, I was also unsure if it'd work, so I tested before commenting to make sure 😅